ci/matrix.yaml

```diff
@@ -53,7 +53,7 @@ workflows:
     - {jobs: ['build'], project: 'cudax', ctk: ['curr'], std: 'all', cxx: ['gcc', 'clang'], cpu: 'arm64'}
     - {jobs: ['test'],  project: 'cudax', ctk: ['curr'], std: 20,    cxx: ['gcc', 'clang', 'msvc'], gpu: 'rtx2080'}
     # Python and c/parallel jobs:
-    - {jobs: ['test'], project: ['cccl_c_parallel'], gpu: ['rtx2080', 'h100']}
+    - {jobs: ['test'], project: ['cccl_c_parallel'], gpu: ['rtx2080', 'l4', 'h100']}
```

Comment:
Can you move these (and the python jobs) to `t4`? The L4 queue is heavily contested and we have some spare capacity on t4, with more on the way.